bundles/http/Citrix/c8afddd5-e247-4184-bc65-8c8a88b824c2/i18n/ja.json

```diff
@@ -0,0 +1,145 @@
+{
+  "c31de16d-990b-400a-83ff-ddc5c53ef9bd": {
+    "7ab7eb5a-2606-4c72-995a-d0b77a068673": "Location",
```

Comment:
Unlocalized.

Reply:
@yangwa-ctx this is a test PR. actual translations are in progress.